apps/src/templates/PaneHeader.jsx

```diff
@@ -41,6 +41,15 @@ var styles = {
     marginLeft: 3,
     marginRight: 0
   },
+  headerButtonMinecraft: {
+    backgroundColor: '#606060',
+    color: '#BFBFBF',
```

Comment:
These colors are not currently in color.scss. Not sure if we want them to (they're MC only) or what I should call them if we do want them to be.